src/planscape/projects/migrations/0002_auto_20240425_1845.py

```diff
@@ -0,0 +1,82 @@
+import json
+from pathlib import Path
+from django.db import migrations
+from django.conf import settings
+from projects.models import ProjectVisibility
+from django.contrib.gis.geos import GEOSGeometry, MultiPolygon
+
+
+def get_geometry(region_name):
+
+    filename = f"projects/migrations/{region_name}.geojson"
```

Comment:
super nit here, and maybe the regions geojson should live somewhere else (fixtures?) given that the files are not migrations themselves?